src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +25,23 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryDao countryDao = new CountryDaoImpl(sessionFactory);
+        CountryService countryService =
+                new CountryServiceImpl(countryDao); // TODO: initialize this instance
```

Comment:
```suggestion
                new CountryServiceImpl(countryDao);
```
you are resolve this TODO, so you should  remove it